spec/takeaway_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:order_klass) { double :order_klass, new: menu_klass, calculate_quantities: order, calculate_cost: order, calculate: order }
```

Comment:
Metrics/LineLength: Line is too long. [130/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)